### PR TITLE
LPD-16619 Get portalURL properly and mock servlet request if needed to fix integration test

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related ticket: [LPD-16619](https://liferay.atlassian.net/browse/LPD-16619)

### CODE REVIEW

**Review by G-Sa — APPROVED** [review]

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java [hunk 1]
```diff
@@ -131,14 +137,29 @@ public static void setUpClass() throws Exception {
 		).put(
 			"${portalURL}",
 			() -> {
-				Company company = _companyLocalService.getCompany(
-					TestPropsValues.getCompanyId());
+				HttpServletRequest httpServletRequest =
+					_originalHttpServletRequest =
```
Comment (marcelabc) [inline modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java:141]: You are assignment a variable and initializing the variable in the same line. This is possible but I've never seen this in Liferay. Could you show me an example, please?
I would prefer to split this code into multiple lines so you can assign the variable and initialize it afterward. What do you think?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java (hunk 1 above)
Comment (devpedrotavares) [reply modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java]: Okay, I did that!

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:object

**Comment by devpedrotavares** [comment]
ci:report:2541383447

**Comment by devpedrotavares** [comment]
ci:forward

**Comment by devpedrotavares** [comment]
ci:forward:force

**Pull request closed without merging**